Main.cc

```diff
@@ -517,6 +532,9 @@ int main(int argc, const char* argv[]) {
             inp.create("root", root_folder, "set top-level folder for data files", "String");
             inp.create("exit_after", "", "number of seconds to stay alive after last sessions exists", "Int");
             inp.create("init_exit_after", "", "number of seconds to stay alive at start if no clents connect", "Int");
+            inp.create("max_payload_length", to_string(max_payload_length), "set uWebSockets max payload length", "Int");
```

Comment:
do these need to be command line arguments? Can't we just use the default ones? Weren't we doing that before?